README.md

```diff
@@ -1,36 +1,53 @@
-</p>
-
-# Liqo
-
 ![Go](https://github.com/liqotech/liqo/workflows/Go/badge.svg) 
 [![Coverage Status](https://coveralls.io/repos/github/liqotech/liqo/badge.svg?branch=master)](https://coveralls.io/github/liqotech/liqo?branch=master)
 [![Go Report Card](https://goreportcard.com/badge/github.com/liqotech/liqo)](https://goreportcard.com/report/github.com/liqotech/liqo)
-![Docker Pulls](https://img.shields.io/docker/pulls/liqo/virtual-kubelet?label=Liqo%20vkubelet%20pulls)
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fliqotech%2Fliqo.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fliqotech%2Fliqo?ref=badge_shield)
 [<img src="https://img.shields.io/badge/slack-liqo.io-yellow">](https://liqo-io.slack.com) 
 [![Twitter](https://img.shields.io/twitter/url/https/twitter.com/liqo_io.svg?style=social&label=Follow%20%40liqo_io)](https://twitter.com/liqo_io)
 
-Great for:
-* Dynamic resource sharing across clusters, with decentralized governance
-* Seamless multi-cluster management
-* Edge clusters orchestration
-
-Liqo is in an alpha version and not yet production-ready.
-
-Community Meeting: Monday 18.30 CET, 9.30 PST [here](https://meet.google.com/dyr-ieso-smu). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=18:30&tz=CET%20%28Central%20European%20Time%29)
-
-# What is Liqo?
+<p align="center">
+  
+  <a href="https://github.com/liqotech/liqo">
+    <img alt="Liqo Logo" src="https://doc.liqo.io/images/logo-liqo-blue.svg" height="80">
+  </a>
+
+  <h3 align="center">A <a href=https://github.com/kubernetes/kubernetes>Kubernetes</a> platform about dynamic and decentralized multi-cluster</h3>
+
+  <br />
+
+  <p align="center">
+    <a href="https://doc.liqo.io/"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://www.youtube.com/watch?v=tHCKGwnmuAA&t=1s&ab_channel=Liqo">View Demo</a>
+    ·
+    <a href="https://github.com/liqotech/liqo/issues/new?assignees=&labels=&template=bug_report.md&title=">Report Bug</a>
+    ·
+    <a href="https://github.com/liqotech/liqo/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%5BFeature%5D">Request Feature</a>
+  </p>
+</p>
 
-Liqo is a framework to enable dynamic sharing across Kubernetes Clusters. You can run your pods on a remote cluster
-seamlessly, without any modification (Kubernetes or your application). 
+## About the project
+Liqo is a platform to enable dynamic and decentralized resource sharing across Kubernetes clusters. Liqo allows to run pods on a remote cluster seamlessly and without any modification of Kubernetes and the applications. With Liqo it is possible to extend the control plane of a Kubernetes cluster across the cluster's boundaries, making multi-cluster native and transparent: collapse an entire remote cluster to a virtual local node, by allowing workloads offloading and resource management compliant with the standard Kubernetes approach.
+
```

Comment:
I would add here the other distinguishing features that we have: Dynamic Discovery, Built-in cluster interconnection, and multi-cluster services

Reply:
Yes, I wanted to describe the Liqo features in a proper section main features